src/overlays/actors/ovl_En_Test3/z_en_test3.c

```diff
@@ -1098,7 +1098,7 @@ s32 EnTest3_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f*
     EnTest3* this = THIS;
 
     if (limbIndex == KAFEI_LIMB_ROOT) {
-        D_80A41D6C = &this->player.bodyPartsPos[-1];
+        D_80A41D6C = this->player.bodyPartsPos - 1;
```

Comment:
```suggestion
        D_80A41D6C = &this->player.bodyPartsPos[0] - 1;
```
I feel like this is easier to read since it shows it's pointer math? (Same for `sPlayerCurBodyPartPos`)

Reply:
Also tempted to ask to name `D_80A41D6C` -> `sKafeiCurBodyPartPos` to parallel `sPlayerCurBodyPartPos` but you don't have to if you don't want to since it's out of scope